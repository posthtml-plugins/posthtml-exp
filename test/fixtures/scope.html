<p>x</p>
<scope with="data">
  <p>Hi. I am {{name}}. I am {{age}} years old.</p>
</scope>
<p>x</p>
