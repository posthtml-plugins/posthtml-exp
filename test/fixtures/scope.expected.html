<p>x</p>

  <p>Hi. I am John. I am 26 years old.</p>

<p>x</p>
